interface/basisu_c_binding/inc/basisu_c_binding.h

```diff
@@ -4,36 +4,32 @@
 #pragma once
 
 #if defined(KHRONOS_STATIC)
-  #define KTX_BASISU_API
+#define KTX_BASISU_API
```

Comment:
I prefer to keep the PP directives indented. I see from fmt's .clang_format that it is possible:

`IndentPPDirectives: BeforeHash`

will do it.

Reply:
Added.